<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>490</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Lista Grupos Laboratorio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>793</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Asignación</string>
     </attribute>
     <widget class="QLabel" name="Creditos">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>370</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TFG - Andrea Magro Canas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSecundario">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pulsa</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnPrincipal">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pulsa</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Titulo_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>761</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>ASIGNACIÓN DE ESTUDIANTES</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="Linea_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>721</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Disponibilidad</string>
     </attribute>
     <widget class="QTreeWidget" name="AsignaturasTree">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>256</width>
        <height>321</height>
       </rect>
      </property>
      <property name="autoExpandDelay">
       <number>-1</number>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>ELECCION DE ASIGNATURAS</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Primer Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Algebra Lineal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Calculo Infinitesimal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fisica I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quimica</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Segundo Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Ampliacion de Matematicas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Estadistica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fisica II</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Metodos Numericos</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Tercer Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Ciencia de materiales</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Resistencia de los materiales</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teoria de Circuitos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Termodinamica</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Cuarto Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Automatica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electronica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maquinas electricas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mecanica de fluidos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tecnologias de fabricacion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Transmision de calor</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Quinto Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Electronica Analogica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electronica Digital</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regulacion Automatica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teoria de maquinas y mecanismos</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Sexto Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Automatizacion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electronica de Potencia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Informatica Industrial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Instrumentacion electronica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Robotica</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Septimo Cuatrimestre</string>
       </property>
       <item>
        <property name="text">
         <string>Control</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SII</string>
        </property>
       </item>
      </item>
     </widget>
     <widget class="QLabel" name="Titulo_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>12</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DISPONIBILIDAD DE LAS ASIGNATURAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="ComboBoxDias">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <item>
       <property name="text">
        <string>Lunes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Martes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Miércoles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jueves</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Viernes</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="ComboBoxHoras">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxVisibleItems">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>09:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17:30</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblHoras">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HORAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblDias">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>DÍAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="Linea_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>721</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnGuardaHorario">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAsignada">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Asignatura asignada:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAsignaturaAsignada">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Agregar Labs</string>
     </attribute>
     <widget class="QSpinBox" name="PlazasText">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>101</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="NumSubgruposText">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="NumSesionesText">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="SemanaInicialText">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblAsignatura">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ASIGNATURA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblNumPlazas">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nº PLAZAS/SESIÓN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblNumSesiones">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nº SESIONES</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblNumSubgrupos">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nº SUBGRUPOS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblSemanaInicial">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SEMANA INICIAL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Titulo_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>12</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AGREGAR LABORATORIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAceptar">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>120</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnVaciarTxt">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Borrar Laboratorios</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="ComboBoxAsignatura">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblHorario">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HORARIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="Linea_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>721</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>189</width>
         <height>199</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>BtnPrincipal</tabstop>
  <tabstop>BtnSecundario</tabstop>
  <tabstop>AsignaturasTree</tabstop>
  <tabstop>ComboBoxDias</tabstop>
  <tabstop>ComboBoxHoras</tabstop>
  <tabstop>BtnGuardaHorario</tabstop>
  <tabstop>ComboBoxAsignatura</tabstop>
  <tabstop>PlazasText</tabstop>
  <tabstop>NumSesionesText</tabstop>
  <tabstop>NumSubgruposText</tabstop>
  <tabstop>SemanaInicialText</tabstop>
  <tabstop>BtnAceptar</tabstop>
  <tabstop>BtnVaciarTxt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
